app/controllers/home/application_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Home::ApplicationController < ActionController::Base
+  rescue_from ActiveRecord::RecordNotFound, with: :not_found
+
+  private
+
+  def not_found
+    flash[:alert] = 'Вы обратились к несуществующей записи.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.